src/frontend/src/flows/addDevice/manage/addDevicePickAlias.ts

```diff
@@ -13,6 +13,8 @@ const pageContent = () => html`
         placeholder="Device alias"
         type="text"
         required
+        maxlength="30"
+        pattern="^[A-Za-z0-9_]+((-|\\s)*[A-Za-z0-9_])*$"
```

Comment:
do you think it'd be worth having a comment here explaining what this matches? maybe an example or two? I always find it hard to tell with confidence what a regex actually matches